.github/dependabot.yml

```diff
@@ -11,6 +11,8 @@ updates:
       - "julienw"
       - "canova"
       - "gregtatum"
+    allow:
+      - dependency-type: "direct"
```

Comment:
I'd like that we get updates only for the direct dependencies, to reduce a bit the load...

According to the doc this has no effect on npm-based projects thought, so maybe this will do nothing and I'll remove it next time.

Reply:
Note that according to the doc, transitive dependencies will still be checked for security issues.

Reply:
Hum actually I think they already track direct dependecies only... i'll remove this.